app/assets/stylesheets/groups.scss

```diff
@@ -20,3 +21,48 @@
 .floatRight {
   float: right;
 }
+
+.toolbar {
+  display: none;
+}
+
+#viewerContainer{
```

Comment:
I have already asked you to put spaces before `{`. Please do this in the future so that I do not need to repeat myself.